docs/installation/Installing_ACME_Responder.md

```diff
@@ -0,0 +1,189 @@
+
+This policy is needed by the ACME profile, but currently it is not installed by default in the CA,
+so it has to be added manually.
+
+To add the policy, edit /etc/pki/pki-tomcat/ca/registry.cfg as follows:
+
+```
+defaultPolicy.ids=...,sanToCNDefaultImpl
+defaultPolicy.sanToCNDefaultImpl.class=com.netscape.cms.profile.def.SANToCNDefault
+defaultPolicy.sanToCNDefaultImpl.desc=SAN to CN Default
+defaultPolicy.sanToCNDefaultImpl.name=SAN to CN Default
+```
+
+**Note:** Restart the server to enable the policy.
+
+## Installing ACME Profile
+
+The acmeServerCert.cfg is a sample profile for generating server certificates via ACME responder.
+
+This profile is currently not installed by default in the CA, so it needs to be added and enabled manually.
+
+To add the profile, execute the following command:
+
+```
+$ pki -u caadmin -w Secret.123 ca-profile-add /usr/share/pki/ca/profiles/acmeServerCert.cfg --raw
+```
+
+To enable the profile, execute the following command:
+
+```
+$ pki -u caadmin -w Secret.123 ca-profile-enable acmeServerCert
+```
+
+## Installing ACME Responder
+
+To install the ACME responder on PKI server, execute the following command:
+
+```
+$ pki-server acme-create
+```
+
+The command will create the initial configuration files in /etc/pki/pki-tomcat/acme folder.
+
+## Configuring ACME Responder Database
+
+The database configuration for the ACME responder is located at /etc/pki/pki-tomcat/acme/database.json.
+
+To use an in-memory database, copy the sample configuration file with the following command:
+
+```
+$ cp /usr/share/pki/acme/conf/database/in-memory/database.json \
+    /etc/pki/pki-tomcat/acme/database.json
+```
+
+Alternatively, edit the file as follows:
+
+```
+{
+    "class": "org.dogtagpki.acme.database.InMemoryDatabase"
+}
+```
+
+Currently there are no parameters to configure for in-memory database.
+
+See also [Configuring ACME Responder](https://www.dogtagpki.org/wiki/Configuring_ACME_Responder).
+
+## Configuring ACME Responder Backend
+
+The backend configuration for the ACME responder is located at /etc/pki/pki-tomcat/acme/backend.json.
+
+To use the CA subsystem as the backend for the ACME responder,
+copy the sample configuration with the following command:
+
+```
+$ cp /usr/share/pki/acme/conf/backend/pki/backend.json \
+    /etc/pki/pki-tomcat/acme/backend.json
+```
+
+Alternatively, edit the file as follows:
+
+```
+{
+    "class": "org.dogtagpki.acme.backend.PKIBackend",
+    "parameters": {
+        "url": "https://localhost:8443",
+        "profile": "acmeServerCert",
+        "username": "caadmin",
+        "password": "Secret.123"
+    }
+}
+```
+
+Configure the parameters as needed.
+
+See also [Configuring ACME Responder](https://www.dogtagpki.org/wiki/Configuring_ACME_Responder).
+
+## Deploying ACME Responder
+
+Once everything is ready, deploy the ACME responder with the following command:
+
+```
+$ pki-server acme-deploy
+```
```

Comment:
Can update here as well **pki-server acme-deploy --instance <instance ID>**

Reply:
I've also added a note below to check the `--help` option for more information.